src/plugins/chart/ChartRender.tsx

```
@@ -125,23 +137,129 @@
+    const rest = (config as any)?.rest || {};
+    const legendField: string | undefined = rest?.colorLegend; // 图例维度 → type
+    const groupByField: string | undefined = rest?.groupBy || rest?.categoryField; // 主筛选维度 → category
+    const filterByField: string | undefined = rest?.filterBy; // 二级筛选维度 → filterLable
+
+    return (chartData || []).map((row: any, i: number) => {
+      const rawX = row?.[config?.x as any];
+      const rawY = row?.[config?.y as any];
+
+      // category: 一个表中的不同数据（主筛选）
+      const category =
+        groupByField && row?.[groupByField] != null
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)